wgpu-core/src/command/render.rs

```diff
@@ -474,8 +489,11 @@ pub enum RenderPassErrorInner {
     ResourceUsageConflict(#[from] UsageConflict),
     #[error("render bundle has incompatible targets, {0}")]
     IncompatibleBundleTargets(#[from] RenderPassCompatibilityError),
-    #[error("render bundle has an incompatible read-only depth/stencil flag: bundle is {bundle}, while the pass is {pass}")]
-    IncompatibleBundleRods { pass: bool, bundle: bool },
+    #[error("render bundle has incompatible (depth,stencil) read-only flags: bundle is {bundle:?}, while the pass is {pass:?}")]
+    IncompatibleBundleRods {
+        pass: (bool, bool),
+        bundle: (bool, bool),
+    },
```

Comment:
```suggestion
    #[error("render bundle has incompatible read-only flags: bundle has flags depth = {bundle_depth} and stencil = {bundle_stencil}, while the pass has flags depth = {pass_depth} and stencil = {pass_stencil}. Read-only renderpasses are only compatible with read-only bundles for that aspect.")]
    IncompatibleBundleRods {
        pass_depth: bool,
        pass_stencil: bool,
        bundle_depth: bool,
        bundle_stencil: bool,
    },
```

Just spelling things out a bit more to make sure it's clear what's going on